nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# View posts on homepage

## Description
Created appropriate Post Controller and Repo methods to get the full list of published and approved posts from the server. Implemented this as a PostList on the client side. Minimal styling so far.

Fixes: #3 

[x] This is a new feature

### Testing Instructions
1. Make sure all seed data and configuration is set up with your Firebase data
2. Start the server
3. Run ```npm start``` from the ```Tabloid/client``` folder
4. Log in using a valid username and password
5. You should be redirected to the home page, displaying a list of posts. These should only be the published and approved posts, and should appear in order of publication, newest first. You may need to verify these with your SQL data, as not all that data is rendered to the list itself.

**Note:** the previous ```Hello``` page remains for now. We can delete this later.
